src/application/PackagerController.js

```diff
@@ -25,6 +28,57 @@ class PackagerController extends events.EventEmitter {
     global._PackagerController = this;
   }
 
+  async startOrRestartLocalServerAsync() {
+    if (this._expressServer) {
+      console.log("Waiting for express to close...");
+      await this._expressServer.close();
+      console.log("Closed express; restarting...");
+    }
+
+    let app = express();
```

Comment:
express is fine for now but once koa 2 is out, we should use that everywhere for its better support of async functions and consistency (we use koa elsewhere). it's pretty easy to go from express -> koa.